lib/tenant.js

```diff
@@ -1,23 +1,25 @@
 // @flow
 'use strict'
 
-const apis = {
-  ONEBLINK: 'https://auth-api.blinkm.io',
-  CIVICPLUS: 'https://auth-api.transform.civicplus.com',
+const tenants = {
+  ONEBLINK: {
+    url: 'https://auth-api.blinkm.io',
+    iss:
+      'https://cognito-idp.ap-southeast-2.amazonaws.com/ap-southeast-2_o1t3ntGWx',
+  },
+  CIVICPLUS: {
+    url: 'https://auth-api.transform.civicplus.com',
+    iss: 'https://cognito-idp.us-east-2.amazonaws.com/us-east-2_A92OPccYd',
+  },
 }
```

Comment:
I thought we were updating this to allow for a `ONEBLINK_SDK_ENVIRONMENT` environment variable that we could set to `test` internally when we want to test the SDK?

Reply:
We discussed it